app/main.py

```diff
@@ -1,2 +1,56 @@
+class Node:
+    def __init__(self, key: int | str, value: int | str) -> None:
+        self.key = key
+        self.value = value
+        self.next = None
```

Comment:
why you need it?

Reply:
The Node class is utilized in the Dictionary class to represent individual elements in a hash table, each containing a key-value pair.

Reply:
using a separate Node class allows for more flexibility and control over the data structure. It enables us to store additional information and implement additional functionalities such as extending chains or changing the order of elements. Additionally, utilizing our own Node class provides greater flexibility and control over the data structure.